### PR TITLE
PM2020-11: As Julia, I would like to bulk convert variant products to simple products

### PR DESCRIPTION
**Pull request by mmetayer**
**Description (for Contributor and Core Developer)**

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | OK
| Added legacy Behats               | -
| Added acceptance tests            | -
| Added integration tests           | OK
| Changelog updated                 | OK
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: src/Akeneo/Pim/Enrichment/Bundle/Resources/config/jobs.yml [hunk 1]
```diff
@@ -592,6 +593,18 @@ services:
...
+        arguments:
+            - '%pim_enrich.job.name.convert_to_simple_product%'
+            - '@event_dispatcher'
+            - '@akeneo_batch.job_repository'
+            -
+                - '@pim_enrich.step.convert_to_simple_product.mass_edit'
+        tags:
+            - { name: akeneo_batch.job, connector: '%pim_enrich.connector_name.mass_edit%', type: '%pim_enrich.job.mass_edit_type%' }
+
+
```
Comment (pierallard) [inline src/Akeneo/Pim/Enrichment/Bundle/Resources/config/jobs.yml:607]: :fire:

File: src/Akeneo/Pim/Enrichment/Bundle/Resources/translations/messages.en_US.yml [hunk 2]
```diff
@@ -83,6 +83,10 @@ pim_enrich:
             success_flash: Attribute requirements were applied to the selected families
             launched_flash: The bulk action "set attribute requirements" is launched. You will be notified when it will be done.
 
+        convert_to_simple_product:
+            error:
```
Comment (pierallard) [inline src/Akeneo/Pim/Enrichment/Bundle/Resources/translations/messages.en_US.yml:87]: warning?

File: src/Akeneo/Platform/Bundle/InstallerBundle/Resources/fixtures/icecat_demo_dev/jobs.yml [hunk 3]
```diff
@@ -262,6 +262,11 @@ jobs:
         alias:     add_to_existing_product_model
         label:     Add to existing product model
         type:      mass_edit
+    convert_to_simple_product:
+        connector: Akeneo Mass Edit Connector
+        alias:     convert_to_simple_product
+        label:     Convert to simple product
```
Comment (pierallard) [inline src/Akeneo/Platform/Bundle/InstallerBundle/Resources/fixtures/icecat_demo_dev/jobs.yml:268]: convert to simple productS ?

File: src/Akeneo/Platform/Bundle/UIBundle/Resources/public/js/mass-edit/form/product/convert-to-simple-product.ts [hunk 4]
```diff
@@ -0,0 +1,11 @@
+const BaseOperation = require('pim/mass-edit-form/product/operation');
+
+class ConvertToSimpleProduct extends BaseOperation {
+  public render() {
+    this.$el.text('');
```
Comment (pierallard) [inline src/Akeneo/Platform/Bundle/UIBundle/Resources/public/js/mass-edit/form/product/convert-to-simple-product.ts:5]: this.$el.empty();

File: src/Akeneo/Platform/Bundle/UIBundle/Resources/public/js/mass-edit/form/product/convert-to-simple-product.ts [hunk 5]
```diff
@@ -0,0 +1,11 @@
+const BaseOperation = require('pim/mass-edit-form/product/operation');
+
+class ConvertToSimpleProduct extends BaseOperation {
+  public render() {
```
Comment (pierallard) [inline src/Akeneo/Platform/Bundle/UIBundle/Resources/public/js/mass-edit/form/product/convert-to-simple-product.ts:4]: public render(): ConvertToSimpleProduct {

**Review by pierallard — APPROVED** [review]

**Review by nmarniesse — APPROVED** [review]

**Pull request merged**